.travis.yml

```diff
@@ -15,5 +15,15 @@ env:
   - PUPPET_VERSION="~> 4.1.0"
   - PUPPET_VERSION="~> 4.2.0"
   - PUPPET_VERSION="~> 4.3.0"
+matrix:
+  include:
+    - rvm: '2.1'
+      sudo: required
+      services: docker
+      env: PUPPET_VERSION="3.8.4" BEAKER_set="centos-6-docker"
+      script: bundle exec rspec spec/acceptance/class_spec.rb
```

Comment:
I was mainly worried about the time it would take, but I can change it to run everything if required? :smile: